### PR TITLE
Lazy load routes 

### LINKED ISSUE #1105: Lazy load routes + refactor
**Description**
Lazy load routes + refactor

#### Definition of done
- [x] Description is created (Task creator)
- [x] Code is complete (Developer)
- [x] Scope of test and what was done section is defined in task (Developer)
- [x] Review passed (Code reviewer)
- [ ] Verified manually (QA)
- [ ] Test cases created/updated if necessary (QA)
- [ ] Change successfully merged
- [ ] Release notes added if needed

**Proposal**

**Additional context**
Make each route lazy load to split js into chunks for faster first page load.

### PR DESCRIPTION
**Pull request by wixzi**
# Summary

Fixes #1105 

Split each route to separate chunk and prefetch it. Expected to load the site faster since the initial chunk is now much smaller.

  # To Test

1. All routes should work as expected
2. All tests should pass

## TODO
- [ ] Need to do a webpack code analysis and split the packages in a better way. (Will do in another ticket) 
- [ ] Remove unwanted dependencies

### CODE REVIEW

**Review by Diogomartf — APPROVED** [review]

File: src/pages/Routes.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import { useActiveWeb3React } from '../hooks'
+import { chainSupportsSWPR } from '../utils/chainSupportsSWPR'
+import { RedirectDuplicateTokenIds, RedirectOldAddLiquidityPathStructure } from './AddLiquidity/redirects'
+import { RedirectOldRemoveLiquidityPathStructure } from './RemoveLiquidity/redirects'
+// Dont want to lazy import swap
+import Swap from './Swap'
+import { RedirectToSwap } from './Swap/redirects'
+
+// Lazy loaded routes
+const Rewards = lazy(() => import(/* webpackPrefetch: true */ './Rewards'))
```
[the remark this answers is not in the document]
Comment (wixzi) [reply src/pages/Routes.tsx:13]: https://webpack.js.org/guides/code-splitting/#prefetchingpreloading-modules

File: src/hooks/useParsedQueryString.ts [hunk 2]
```diff
@@ -1,13 +1,7 @@
...
 
-// TODO: qs package can be removed and replaced with useSearchParams from react-router-dom
-// Object.fromEntries(searchParams) gives the same result
-export default function useParsedQueryString(): ParsedQs {
-  const { search } = useLocation()
-  return useMemo(
-    () => (search && search.length > 1 ? parse(search, { parseArrays: false, ignoreQueryPrefix: true }) : {}),
-    [search]
-  )
+export default function useParsedQueryString() {
```
Comment (adamazad) [inline src/hooks/useParsedQueryString.ts:4]: Let's remove default exports :)

File: src/components/Loader/FallbackLoader.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  return (
+    <Wrapper fill={fill} height={height}>
+      <AnimatedImg>
+        <img src={Logo} alt="loading-icon" />
+      </AnimatedImg>
+    </Wrapper>
+  )
+}
+
+export default FallbackLoader
```
Comment (adamazad) [inline src/components/Loader/FallbackLoader.tsx:53]: Named exports please

File: src/pages/Routes.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+const RouteWrapper = ({ element }: { element: JSX.Element }) => {
+  const { chainId } = useActiveWeb3React()
+  // If all features are available, render the route
+  if (chainSupportsSWPR(chainId)) {
+    return element
+  }
+  return <RedirectToSwap />
+}
+
+export default function Routes() {
```
Comment (adamazad) [inline src/pages/Routes.tsx:35]: Here

File: src/pages/Routes.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+const MyPairs = lazy(() => import(/* webpackPrefetch: true */ './Pools/Mine'))
+const Bridge = lazy(() => import(/* webpackPrefetch: true */ './Bridge'))
+const AddLiquidity = lazy(() => import(/* webpackPrefetch: true */ './AddLiquidity'))
+const RemoveLiquidity = lazy(() => import(/* webpackPrefetch: true */ './RemoveLiquidity'))
+const CreateLiquidityMining = lazy(() => import(/* webpackPrefetch: true */ './LiquidityMining/Create'))
+
+/**
+ * A Route that is only accessible if all features available: Swapr core contract are deployed on the chain
+ */
+const RouteWrapper = ({ element }: { element: JSX.Element }) => {
```
Comment (adamazad) [inline src/pages/Routes.tsx:26]: Much cleaner :) 👏

**Review by adamazad — APPROVED** [review]
Great work as usual :)

**Review by berteotti — APPROVED** [review]

**Pull request merged**